src/output/icons.rs

```diff
@@ -173,9 +179,10 @@ const FILENAME_ICONS: Map<&'static str, char> = phf_map! {
     ".npmrc"              => Icons::NPM,            // 
     ".profile"            => Icons::SHELL,          // 󱆃
     ".python_history"     => Icons::LANG_PYTHON,    // 
-    ".rustfmt.toml"       => Icons::LANG_RUST,      // 
+    ".rustfmt.toml"       => Icons::LANG_RUST,      // 
     ".rvm"                => Icons::LANG_RUBY,      // 
     ".rvmrc"              => Icons::LANG_RUBY,      // 
+    "README"              => Icons::README,         // 󰂺
```

Comment:
Just noticed this is not sorted properly.  You can fix or I can fix the next time I edit the icons.